packages/client/src/utils/getTestClient.ts

```diff
@@ -50,7 +49,7 @@ export async function getTestClient(schemaDir?: string, printWarnings?: boolean)
   const outputDir = absSchemaDir
   const relativeEnvPaths = getEnvPaths(schemaPath, { cwd: absSchemaDir })
   const activeProvider = config.datasources[0].activeProvider
-  const options: GetPrismaClientConfig = {
+  const options = {
```

Comment:
Was the type annotation removed just because it wasn't necessary or because it was causing some problems?

Reply:
I changed the reference to runtime module from `../runtime/` (`client/src/runtime`) to `../../runtime/` (`client/runtime`) in the above lines, and `client/runtime` doesn't export `GetPrismaClientConfig`. So I removed `GetPrismaClientConfig`. (also it doesn't look absolutely necessary)

I changed the reference to runtime like this because if we import from `client/src/runtime/` it throws require error at `sql-template-tag` because of ESM issue. The built version of runtime is in CommonJS, so it can avoid that issue. I'm not 100% sure this approach is acceptable or not. What do you think?

Reply:
Ah, this change seems causing the mismatch of query engine version in CI

I now see these errors..
```
Query engine library for current platform "debian-openssl-1.1.x" could not be found.
    You incorrectly pinned it to debian-openssl-1.1.x

    This probably happens, because you built Prisma Client on a different platform.
    (Prisma Client looked in "/home/runner/work/prisma/prisma/packages/client/runtime/libquery_engine-debian-openssl-1.1.x.so.node")
```

Reply:
It could have been a viable option to call `generateInFolder` instead, that might have solved the issue. I feel like using the runtime as it is shipped (like you proposed) is a viable option. The tests in `__tests__` are currently being upgraded to a cleaner and more standard setup, so it is only a matter of time until we remove them. Additionally, swapping `getPrismaClient` to `generateInFolder` should be transparent.

Reply:
(see https://github.com/prisma/prisma/pull/14589#discussion_r947135925)